rudder-jetty/SPECS/rudder-jetty.spec

```diff
@@ -177,6 +177,17 @@ then
 	%endif
 fi
 
+%preun -n rudder-jetty
+#=================================================
+# Pre Un-installation
+#=================================================
+
+if [[ $1 -eq 0 ]]
+then
+  service rudder-jetty stop
+fi
```

Comment:
Note: this does not need doing on debian, because dh_installinit automaticaly does this for us.